api/apps/api/test/users.e2e-spec.ts

```diff
@@ -434,4 +434,44 @@ describe('UsersModule (e2e)', () => {
       expect(WhenGettingAdminListResponse.body[0].userId).toEqual(adminUserId);
     });
   });
+
+  describe('Users - Find by email', () => {
+    let adminToken: string;
+    let adminUserId: string;
+    const cleanups: (() => Promise<void>)[] = [];
+
+    beforeAll(async () => {
+      adminToken = await GivenUserIsLoggedIn(app, 'dd');
+      adminUserId = await GivenUserExists(app, 'dd');
+    });
+
+    afterEach(async () => {
+      for (const cleanup of cleanups.reverse()) {
+        await cleanup();
+      }
+    });
+
+    test('A user can be found by full email', async () => {
+      const userId = await GivenUserExists(app, 'aa');
+      const WhenSearchingUserByFullMail = await request(app.getHttpServer())
+        .get('/api/v1/users/by-email')
+        .set('Authorization', `Bearer ${adminToken}`)
+        .send({ email: 'aa@example.com' });
+      expect(WhenSearchingUserByFullMail.status).toEqual(200);
+      expect(WhenSearchingUserByFullMail.body.data).toBeDefined();
+      expect(WhenSearchingUserByFullMail.body.data.id).toEqual(userId);
+      expect(
+        WhenSearchingUserByFullMail.body.data.attributes.displayName,
+      ).toEqual('User A A');
+    });
+
+    test('Partial email searches return undefined', async () => {
+      const WhenSearchingUserByFullMail = await request(app.getHttpServer())
+        .get('/api/v1/users/by-email')
+        .set('Authorization', `Bearer ${adminToken}`)
+        .send({ email: 'aa@example' });
```

Comment:
what if the API user sent us `{ email: 'aa@example%' }` though? 😉 

I suggest to create a new method such as `UsersService.findByExactEmail()`, or replace the existing `findByEmail()` method altogether, in either case dropping the `ILike` operator - basically all we should do in that service function is `'... where lower(email) = :email', { email: email.toLowerCase() }` or something like that.